pyproject.toml

```diff
@@ -1,3 +1,6 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
```

Comment:
The LICENSE file indicates 2022. Should it be synced then?

Reply:
Let's bump the year in the license file up to 2023 if I understand your comment correctly. If I am not mistaken, the new Apache license file was created in 2023.

Reply:
Bumped the license year in https://github.com/omnivector-solutions/slurmdbd-operator/pull/1/commits/27275512c75b1018dfdc1cd375699933a96cca89